var/spack/repos/builtin/packages/r-data-table/package.py

```diff
@@ -34,3 +37,4 @@ class RDataTable(RPackage):
     version('1.9.6',    sha256='6f74c349c1731823aef6899edcf18418454167d04eba983e3a6fe17ee9fd236e')
 
     depends_on('r@3.1.0:', type=('build', 'run'))
+    depends_on('zlib', type=('build', 'run'))
```

Comment:
Is this the correct type of dependency for a library?

Reply:
No, it is not. Looks like I did a opy/paste and dir not complete the edit. I just pushed a new commit.

Thanks.